src/Funds/FundDetails/FundDetailsContainer.js

```diff
@@ -67,9 +75,25 @@ const FundDetailsContainer = ({
           return mutator.fundCurrentFY.GET({
             path: `${LEDGERS_API}/${fundResponse.fund.ledgerId}/current-fiscal-year`,
           });
+        }, () => {
+          showToast('ui-finance.fund.actions.load.error', 'error');
+
+          setCompositeFund({ fund: {}, groupIds: [] });
+        })
+        .then(currentFYResponse => {
+          setCurrentFY(currentFYResponse);
+
+          return mutator.currentBudget.GET({
+            params: {
+              query: `fundId==${params.id} and fiscalYearId==${currentFYResponse.id}`,
+            },
+          });
+        }, () => {
+          showToast('ui-finance.fiscalYear.actions.load.error', 'error');
```

Comment:
it's better to `setCurrentFY()` here since previous will stay, what do you think?

Reply:
exactly right 👍